golang/pkg/cli/display.go

```diff
@@ -52,3 +59,61 @@ func NotifyOnce(name string, notifyFunc func()) {
 	}
 	notifyFunc()
 }
+
+type status struct {
+	Current int64
+	Total   int64
+}
+
+func DockerPullProgressDisplayer(header string, respIn io.ReadCloser) error {
+	if respIn == nil {
+		log.Info().Msgf("%s ✓ up-to-date", header)
+		return nil
+	}
+
+	dec := json.NewDecoder(respIn)
+	stat := map[string]*status{}
```

Comment:
I'm not a go hero, which is the preferred way? 

```stat := make(map[string]*status)```

Reply:
I don't like make, they are equivalent.